src/EFCore.PG/Migrations/Internal/NpgsqlMigrationsAnnotationProvider.cs

```diff
@@ -60,7 +60,15 @@ public override IEnumerable<IAnnotation> For(IIndex index)
             if (index.GetNullSortOrder() is IReadOnlyList<SortOrder> nullSortOrder)
                 yield return new Annotation(NpgsqlAnnotationNames.IndexNullSortOrder, nullSortOrder);
             if (index.GetIncludeProperties() is IReadOnlyList<string> includeProperties)
-                yield return new Annotation(NpgsqlAnnotationNames.IndexInclude, includeProperties);
+            {
+                var includeColumns = (IReadOnlyList<string>)includeProperties
```

Comment:
Any reason to cast a value? `Annotation` isn't generic, so the value will be stored as an `object` in any case.

Reply:
Not really, just copying and pasting code from EF Core. Will remove the cast.